dist/index.html

```diff
@@ -124,8 +124,70 @@ <h3 class="text-c100 text-md py-3 font-bold">We Nourish</h3>
       Peoples Feedback
     </section>
 
-    <section class="work-style">
-      Work Style
+    <section class="work-style text-c600">
+      <div class="container">
+        <h2 class="text-center text-c100 font-bold">
+          How We <span class="text-c200 font-hairline underline">Work</span>
+        </h2>
+
+        <div class="showcase-row items-center">
+          <div class="text-center">
+            <div class="border-solid border-6 rounded-full border-c200">
+              <img
+                class="mx-auto p-8"
+                src="./assets/img/work-style/01.png"
+                alt=""
+              />
+            </div>
+            <h3 class="showcase-row__heading">Join Our Website</h3>
+            <p class="text-sm py-2">
+              Lorem ipsum is simply free text available in market websites
+            </p>
+          </div>
+
+          <div class="text-center ml-10 pl-10">
+            <div class="h-1/2">
+              <img
```

Comment:
you need to add the border styles in the parent div instead of the image here

and add a specific width and height to them

Reply:
Thanks that was helpful